tracing/spanWrapper.go

```diff
@@ -0,0 +1,12 @@
+package tracing
+
+import (
+	ot "github.com/opentracing/opentracing-go"
+	opentelemetry "go.opentelemetry.io/otel/trace"
+)
+
+type SpanWrapper struct {
+	Ot      ot.Span
+	Otel    opentelemetry.Span
+	useOtel bool
+}
```

Comment:
Implement all methods here to dispatch which implementation to use. This should not leak to all the other packages.